examples/src/bin/create_sub_organization.rs

```diff
@@ -0,0 +1,111 @@
+use reqwest::{
+    header::{HeaderMap, HeaderValue},
+    StatusCode,
+};
+use serde_json::json;
+use std::error::Error;
+use std::{env, time};
+use tkhq_api_key_stamper::{stamp, TurnkeyApiKey};
+
+// See <https://docs.turnkey.com/api-reference/organizations/create-sub-organization> for documentation
+const TURNKEY_API_HOST: &str = "https://api.turnkey.com";
+const WHOAMI_PATH: &str = "/public/v1/submit/create_sub_organization";
```

Comment:
Let's update to `CREATE_SUB_ORGANIZATION_PATH`

Reply:
Oh whoops. Good catch!